website/docs/outposts/index.mdx

```diff
@@ -2,39 +2,67 @@
 title: Outposts
 ---
 
-An outpost is a single deployment of an authentik component, which can be deployed in a completely separate environment:
+An outpost is a single deployment of an authentik component, essentially a service, that can be deployed anywhere that allows for a connection to the authentik API.
+
+An outpost is required if you use any of the following types of providers with your application:
 
 -   [LDAP Provider](../providers/ldap/index.md)
 -   [Proxy Provider](../providers/proxy/index.md)
 -   [RADIUS Provider](../providers/radius/index.md)
 -   [RAC Provider](../providers/rac/index.md)
 
-![](outposts.png)
+These types of providers use an outpost for increased flexibility and speed. Instead of the provider interacting directly with the business logic in the authentik Core to process all transactions, these providers use an outpost to handle the logic, which provides better isolation and performance improvements (because the outpost is written in Go while the provider is written in Python).
```

Comment:
```suggestion
These types of providers use an outpost for increased flexibility and speed. Instead of the provider logic being implemented in authentik Core, these providers use an outpost to handle the logic, which provides improved performance.
```